src/components/CardGroup/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ChakraProps, SimpleGrid } from '@chakra-ui/react'
+import { Card } from '@components'
+
+interface CardGroupProps extends ChakraProps {
```

Comment:
`ChakraProps` has no effect here. 

When this can make sense? If you want to modify `Card` component with any arbitrary chakra prop  (this is not necessary actually) you need to pass `...rest` props like below:

```tsx
interface CardGroupProps extends ChakraProps {
  items: SubpageType[]
  isSimple: boolean
}

export const CardGroup = ({ items, isSimple, ...rest }: CardGroupProps): JSX.Element => {
  return (
    <SimpleGrid columns={{ base: 1, md: 2, lg: 4 }} spacing={10}>
      {items.map((item, i) => (
        // rest contains all ChakraProps
        <Card key={i} item={item} isSimple={isSimple} {...rest} />
      ))}
    </SimpleGrid>
  )
}
```

Let's say you want to modify `Card` component from CardGroup:

```tsx
...
<CardGroup items={data} isSimple={false} borderWidth={1} borderColor="primary.400" />
...
```